src/Telegram/Types/Message.php

```diff
@@ -417,4 +418,16 @@ protected function mapSubObjects(string $key, array $data): TelegramTypes
         // Return always null if none of the objects above matches
         return parent::mapSubObjects($key, $data);
     }
+
+    /**
+     * Decode style entities from Telegram bot messages (bold, italic, etc.) in text with inline entities that duplicate (when possible) the exact style the message had originally when was sended to the bot.
+     *
+     * @param string $style
+     * @return string
+     */
+    public function decodeEntities($style = 'HTML')
+    {
+        $entitiesDecoder = new EntityDecoder($style);
```

Comment:
What happens if the passed style is invalid? Does it throw an exception? If so, add a `@throws` comment above or perform some kind of validation here. 

Ideally, you library should handle this but I saw you don't throw anything on the constructor here: https://github.com/LucaDevelop/telegram-entities-decoder/blob/master/src/EntityDecoder.php#L30